saltstack/final/salt/cis-controls/scripts/cis_control.sh

```diff
@@ -43,7 +43,7 @@ if [ "${CLOUD_PROVIDER}" == "Azure" ]; then
 fi
 
 #Install and download what we need for the hardening
-python3 -m virtualenv $ANSIBLE_PATH
+python3 -m virtualenv --python="/usr/bin/python3.11" $ANSIBLE_PATH
```

Comment:
I think we have a python3.11 wrapper so it could be simplified
`python3.11 -m virtualenv $ANSIBLE_PATH`

Reply:
I hope you don't mind, but I won't change it at this point. After some trial and error it turned out that neither Python 3.11 and 3.9 works with the CIS Playbook so wasted like 4-5 hours already on finding out that I'll have to use Python 3.8. This is reflected in the latest force-push and I'd rather not make another round with a minimal gain especially because this thing blocks GCP image burning.